docs/platforms/javascript/common/enriching-events/scopes/index.mdx

```diff
@@ -8,35 +8,97 @@ information from the current scope. SDKs will typically automatically manage the
+
+A scope is generally valid inside of a callback or an execution context. This means that multiple parts of your application may have different scopes active at the same time. For instance, a web server might handle multiple requests at the same time, and each request may have different scope data to apply to its events.
+
+## Different Kinds of Scopes
+
+The Sentry SDK has three different kinds of scopes:
+
+- The [global scope](#global-scope)
+- The [isolation scope](#isolation-scope)
+- The [current scope](#current-scope)
+
+### Global Scope
+
+The global scope is applied to _all_ events, no matter where they originate. You can use it to store e.g. environmental information, or other things that should apply to all events.
+
+You can access the global scope via `Sentry.getGlobalScope()`.
+
+### Isolation Scope
+
+The isolation scope is used to isolate events from each other. For example, each request in a web server might get its own isolation scope, so that events from one request don't interfere with events from another request. In most cases, you'll want to put data that should be applied to your events on the isolation scope - which is also why all `Sentry.setXXX` methods, like `Sentry.setTag()`, will write data onto the currently active isolation scope. A classic example for data that belongs on the isolation scope is a user - each request may have a different user, so you want to make sure that the user is set on the isolation scope.
+
+You can access the isolation scope via `Sentry.getIsolationScope()`, but usually you'll just use the `Sentry.setXXX` methods to set data on the currently active isolation scope:
+
+```javascript
+Sentry.setTag("my-tag", "my value");
+// Is identical to:
+Sentry.getIsolationScope().setTag("my-tag", "my value");
+```
+
+<PlatformCategorySection supported={["browser"]}>
+  In the browser, the isolation scope is never forked, because it is impossible
+  to keep track of where an isolation scope would belong to. Because of this, in
+  the browser the isolation scope is effectively global.
+</PlatformCategorySection>
 
-The default SDK integrations will push and pop scopes intelligently. For
-instance web framework integrations will create and destroy scopes around your
-routes or controllers.
+### Current Scope
 
-## How the Scope and Hub Work
+The current scope is the local scope that is currently active. In contrast to the isolation scope, which is usually only forked rarely, e.g. per request, the current scope may be forked more frequently and under the hood. It can be used to store data that should only be applied to specific events. In most cases, you should not access this scope directly, but use `Sentry.withScope` to create a new scope that is only active for a specific part of your code:
```

Comment:
```suggestion
The current scope is the local scope that is currently active. Unlike the rarely-forked isolation scope, the current scope may be forked more frequently and under the hood. It can be used to store data that should only be applied to specific events. In most cases, you should not access this scope directly, but use `Sentry.withScope` to create a new scope that is only active for a specific part of your code:
```